src/std/array.ab

```diff
@@ -26,3 +26,47 @@ pub fun includes(array, value) {
     let result = array_first_index(array, value)
     return result >= 0
 }
+
+/// Returns the first element in the array
+pub fun first(array) {
+    return array[0]
+}
+
+/// Returns the last element in the array
+pub fun last(array) {
+    let index = len(array) - 1
+    return array[index]
+}
+
+/// Removes an element at the index from the array
+pub fun remove_at(ref array: [], index: Num): Null {
+    let offset = index + 1
+    let length = len(array)
+    array = array[0..index] + array[offset..length]
+}
+
+/// Removes an element at the index from the array, and returns it
```

Comment:
If I understand the test cases correctly, passing negative numbers to these functions make the elements not being removed. If so, could be the documentations describe this?

Reply:
When you slice an array by a range, the lower and upper bounds are capped at zero and the array length respectively, so `array[-100..100]` for a 5 element array is the same as `array[0..5]`. It wouldn't hurt to describe this behaviour in the doc comments for these stdlib functions, and I will push a commit for that this evening.

We also need to describe the core functionality in the website documentation; I will work with @Ph0enixKM to get this done.

Reply:
Pushed new commit.